File: CarClick/src/main/java/com/ceiba/adn/dominio/modelo/Cliente.java
package com.ceiba.adn.dominio.modelo;

public class Cliente {
	
	private String idCliente;
	private String nombreCompleto;
	private long telefono;
	private String direccion;
	private String email;
	
	public Cliente(String idCliente, String nombreCompleto, long telefono, String direccion, String email) {
		super();
		this.idCliente = idCliente;
		this.nombreCompleto = nombreCompleto;
		this.telefono = telefono;
		this.direccion = direccion;
		this.email = email;
	}

	public String getIdCliente() {
		return idCliente;
	}

	public void setIdCliente(String idCliente) {
		this.idCliente = idCliente;
	}

	public String getNombreCompleto() {
		return nombreCompleto;
	}

	public void setNombreCompleto(String nombreCompleto) {
		this.nombreCompleto = nombreCompleto;
	}

	public long getTelefono() {
		return telefono;
	}

	public void setTelefono(long telefono) {
		this.telefono = telefono;
	}

	public String getDireccion() {
		return direccion;
	}

	public void setDireccion(String direccion) {
		this.direccion = direccion;
	}

	public String getEmail() {
		return email;
	}

	public void setEmail(String email) {
		this.email = email;
	}

}

File: CarClickFrontEnd/src/app/feature/cliente/shared/model/cliente.ts
export class Cliente {
    idCliente: number;
    nombreCompleto: string;
    telefono: string;
    direccion: string;
    email: string;

    constructor(idCliente: number, nombreCompleto: string, telefono: string, direccion: string, email:string) {
        this.idCliente = idCliente;
        this.nombreCompleto = nombreCompleto;
        this.telefono = telefono;
        this.direccion = direccion;
        this.email = email;
    }
}